rust/arrow/src/array/builder.rs

```diff
@@ -779,140 +781,31 @@ where
         let offset_buffer = self.offsets_builder.finish();
         let null_bit_buffer = self.bitmap_builder.finish();
         let nulls = null_bit_buffer.count_set_bits();
```

Comment:
I just noticed that we had named this unintuitively, as we're counting the set bits

Reply:
`unset`? it proves the point, though 🤣

FYI, @nevi-me , the tests are not running (INFRA team deactivated most actions), so we need to be careful with merging to master. There is a thread in the mailing list about it.

Reply:
Okay, thanks for the heads-up. I'll check my emails